### PR TITLE
Fortinet FortiManager IPv4 Policy Module

### PR DESCRIPTION
**Pull request by ftntcorecse**
##### SUMMARY

Allows the add/set/delete of IPv4 Firewall Policies within Packages in FortiManager.

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

fmgr_fwpol_ipv4.py
test_fmgr_fwpol_ipv4.py
fixtures/test_fmgr_fwpol_ipv4.json
##### ANSIBLE VERSION

```paste below
ansible 2.8.0.dev0 (devel 9efc3dc761) last updated 2018/09/27 20:21:00 (GMT -700)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/usr/ansible_ftnt/library']
  ansible python module location = /root/ansibleSource/ansible/lib/ansible
  executable location = /root/ansibleSource/ansible/bin/ansible
  python version = 2.6.9 (unknown, Sep 27 2018, 21:07:46) [GCC 5.4.0 20160609]
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py [hunk 1]
```diff
@@ -0,0 +1,1510 @@
...
+      - List of multiple child objects to be added. Expects a list of dictionaries.
+      - Dictionaries must use FortiManager API parameters, not the ansible ones listed below.
+      - If submitted, all other prefixed sub-parameters ARE IGNORED. This object is MUTUALLY EXCLUSIVE with its options.
+      - We expect that you know what you are doing with these list parameters, and are leveraging the JSON API Guide.
+      - WHEN IN DOUBT, OMIT THE USE OF THIS PARAMETER AND USE THE SUB OPTIONS BELOW INSTEAD TO CREATE OBJECTS WITH MULTIPLE TASKS
+    required: false
+
+  vpn_dst_node_host:
+    description:
+      - NO DESCRIPTION PARSED ENTER MANUALLY
```
Comment (samccann) [inline lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py]: Starting here, there are a few of these 'NO DESCRIPTION PARSED...' descriptions. I'm guessing you need to manually add something to each of this to describe the parameters?

Comment (ftntcorecse) [reply lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py]: Yep! Sorry I'm adding a #TODO tag in this workflow so we don't miss it in the future.

**Review by samccann — APPROVED** [review]
docs look good.

**Review by NilashishC — APPROVED** [review]

File: lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py [hunk 2]
```diff
@@ -0,0 +1,1510 @@
...
+    HAS_PYFMGR = False
+
+###############
+# START METHODS
+###############
+
+
+def fmgr_firewall_policy_addsetdelete(fmg, paramgram):
+    """
+    fmgr_firewall_policy -- Your Description here, bruh
```
Comment (NilashishC) [inline lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py]: Please modify these generic description statements.

**Review by NilashishC — CHANGES_REQUESTED** [review]
Please modify the generic function description statement.

File: lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py (hunk 2 above)
Comment (ftntcorecse) [reply lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py]: Fixed

**Comment by ftntcorecse** [comment]
@NilashishC - Fixed generic function statement as requested.

**Review by NilashishC — APPROVED** [review]

**Comment by NilashishC** [comment]
LGTM

**Pull request merged**